behat/symfony2-extension/2.1/Makefile

```diff
@@ -0,0 +1,5 @@
+behat-init:
```

Comment:
I don't think this deserves an alias in the Makefile, as you will only run it once in the life of the project (or never, if you write the config files by hand)